api/routes/index.js

```diff
@@ -2,12 +2,8 @@ const express = require("express");
 
 const router = express.Router();
 
-const userRouter = require("./userRouter");
-const itemsRouter = require("./itemsRouter");
-const orderRouter = require("./orderRouter");
+const cartRouter = require("./cartRouter");
 
-router.use("/items", itemsRouter.router);
-router.use("/order", orderRouter.router);
-router.use("", userRouter.router);
+router.use("/cart", cartRouter.router);
```

Comment:
장바구니에 존재하는 데이터는 한 개가 아니므로 아래와 같이 수정해 주세요 !
```suggestion
router.use("/carts", cartRouter.router);
```